src/probability/martingale/basic.lean

```diff
@@ -575,8 +575,8 @@ begin
   have h := set_integral_ge_of_const_le (measurable_set_le measurable_const
     (finset.measurable_range_sup'' (λ n _, (hsub.strongly_measurable n).measurable.le (𝒢.le n))))
     (measure_ne_top _ _) this
-    (integrable.integrable_on (integrable_stopped_value (hitting_is_stopping_time
-     hsub.adapted measurable_set_Ici) hsub.integrable hitting_le)),
+    (integrable.integrable_on (@integrable_stopped_value Ω ℕ _ _ _ _ _ _ _ _ _
+      (hitting_is_stopping_time hsub.adapted measurable_set_Ici) hsub.integrable _ hitting_le)),
```

Comment:
I need to add the `@` because somehow none of the arguments can precisely indicate what the index type is and Lean guesses incorrectly `Ω → ℕ` instead of `ℕ`.